charts/project-origin-registry/templates/registry-deployment.yaml

```diff
@@ -57,6 +74,46 @@ spec:
                   key: {{ required "When type is concordium, accountKeySecret.key must be set" .Values.immutableRecord.concordium.accountKeySecret.key }}
           {{- end }}
 
+          # rabbitmq - operator
+          {{- if .Values.rabbitmq.operatorEnabled }}
+            - name: RabbitMq__Hostname
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Release.Name }}-rabbitmq-default-user
+                  key: host
+            - name: RabbitMq__AmqpPort
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Release.Name }}-rabbitmq-default-user
+                  key: port
+            - name: RabbitMq__HttpApiPort
+              value: {{ required "A valid messageBroker rabbitmq httpPort is required!" .Values.rabbitmq.httpPort | quote }}
+            - name: RabbitMq__Username
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Release.Name }}-rabbitmq-default-user
+                  key: username
+            - name: RabbitMq__Password
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Release.Name }}-rabbitmq-default-user
+                  key: password
```

Comment:
I see you are using a "rabbitmq-default-user", If a default user is used, is the default password used as well?

Reply:
the default-user that is referred to here are generated by the [rabbitmq operator and stored in a secret](https://www.rabbitmq.com/kubernetes/operator/using-operator.html#creds) so they are unique and not some default values people can find online.